src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/course/domain/CourseService.kt

```diff
@@ -8,29 +8,30 @@ import java.util.UUID
 
 @Service
 @Transactional
-class CourseService(
-  @Autowired
-  val courseRepository: CourseRepository,
+class CourseService
+@Autowired
+constructor (
+  private val courseRepository: CourseRepository,
```

Comment:
I think this commit (`a4a7cdd606a` - "Update service layer for naming consistency") should be "Update repository layer for naming consistency", right?